CodeOfConduct.md

```diff
@@ -0,0 +1,101 @@
+* A "free speech" zone. We encourage authentic, curious, respectful conversation, including disagreement, but we do not tolerate all forms of speech. We do not tolerate hate speech of any kind. [Bad-faith](https://en.wikipedia.org/wiki/Bad_faith) debating is equally unwelcome, as is misinformation.
+* A place for general advertisement. Promoting your original work is welcome, but please avoid overly commercial self-promotion
+
+You may [find another server](https://instances.social/) to be a better fit if you don't like these rules.
+
+### Expected Behavior
+
+The following behaviors are expected and requested of all community members:
+
+  * Be friendly, open and welcoming.
+  * Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
+  * Exercise consideration and respect in your speech and actions.
+  * Attempt collaboration & de-escalation before conflict.
+  * Refrain from demeaning, discriminatory, or harassing behavior and speech.
+  * Be mindful of your fellow participants. Alert community leaders if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
+  
+### Unacceptable Behavior
+
+To be specific, the following behaviors are unacceptable within our community:
+
+  * Anything in violation of our [Terms of Service](https://nelson.social/terms) including posting anything illegal in Canada.
+  * Violence, threats of violence or violent language directed against another person.
+  * Using slurs or racist, sexist, homophobic, transphobic, ableist or otherwise discriminatory or hateful jokes or language; promoting white supremacy, anti-Semitism, transphobia, castism or other hateful ideologies.
+  * Posting or displaying sexually explicit or violent material.
+  * Posting or threatening to post other people's personally identifying information ("doxing").
+  * Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
+  * Unwelcome sexual attention. This includes, sexualized comments or jokes; unwelcomed sexual advances.
+  * Deliberate intimidation, stalking or following (online or in person).
+  * Debating moderation actions in public. If you disagree with how a moderator enforced this Code of Conduct you may direct-message the moderation team.
+  * Tone policing. Messages you see on nelson.social may violate the standards of civility we set on nelson.social, but you should not criticize users for their tone. If the user is a member of the nelson.social community, you should report them; if not, you should feel free to block them. In neither case should you criticize their tone.
+  * Advocating for, or encouraging, any of the above behavior.
+
```

Comment:
Would be good to reiterate "if you don't like it, you are very welcome to leave, link to instances.social" here.